playbook/app/pb_kits/playbook/tokens/_titles.scss

```diff
@@ -31,8 +31,19 @@
 
 @mixin pb_title_4 {
   @include pb_title($heading_4, $bolder);
+  letter-spacing: -0.03em !important;
```

Comment:
I'm worried about this important tag. We won't be able to easily override this

Reply:
I removed it in my latest commit. I left it in there by accident and it is not needed. Thanks for catching that.